subsys/bluetooth/host/l2cap.c

```diff
@@ -31,6 +31,7 @@
 
 #define L2CAP_LE_MIN_MTU		23
 #define L2CAP_ECRED_MIN_MTU		64
+#define L2CAP_ECRED_MAX_MTU		65533
```

Comment:
Where is this value coming from, or is this UINT16_MAX - sizeof(uint16_t), afaik there is no such limitation since even if the very first 2 bytes are used for the SDU length that only means that an extra segment with 2 bytes would be needed if the MPS is UINT16_MAX.

Reply:
I browsed through config files some more, shouldn't it just be lower of the values BT_L2CAP_TX_MTU and BT_L2CAP_RX_MTU? Basically the same thing that is done with BT_ATT_MTU.